.buildkite/pipeline.yaml

```diff
@@ -0,0 +1,30 @@
+steps:
+  - command:
+    - "./build.sh"
+    label: "\U0001F528 Build - Go 1.12"
```

Comment:
the emoji have names - it would be nice to use them.

See https://github.com/buildkite/emojis

Reply:
We can't as they clutter up the PR "Checks" display I'm afraid.